### ISSUE TITLE
Publish 0.7.0

### ISSUE THREAD

**Issue by kylebarron** [kylebarron/parquet-wasm#583]
cc @H-Plus-Time in case there's anything you want to get into 0.7.0.

**Comment by H-Plus-Time** [comment]
Yep, I think at this point #545 and #529 , I should be able to solve those in one hit (all in the module exports). Maybe closing out the documentation one for async iterable polyfill too.

**Comment by lm93547** [comment]
Bump?

**Comment by kylebarron** [comment]
@lm93547 It looks like you care more about geoparquet-wasm instead? That's not distributed by this repo, that's in
